src/index.js

```diff
@@ -0,0 +1,19 @@
+import top from "./top/top";
+import homepage from "./homepage/homepage";
+import menu from "./menu/menu";
+import buttonSection from "./buttonSection";
+const content = document.getElementById("content");
+function renderPage(toRender) {
+  const container = document.createElement("div");
+  console.log(top());
+  container.appendChild(top());
+  container.appendChild(buttonSection());
+  if (toRender === "homepage") container.appendChild(homepage());
```

Comment:
better way to write this will be:

```
const sectionToRender = (section === "homepage") ? homepageSection() : menuSection();
container.appendChild(sectionToRender);
```